### DISCUSSION TITLE
Incorrect focus on a popup when deleting a repository

### DISCUSSION THREAD

**Discussion by avganka** [community/community#55553] [Accessibility]
### Select Topic Area

Bug

### Body

When opening a popup to enter the name of a repo to delete it, the focus does not move into the input. If you start typing immediately after opening the popup, different actions on the page start to occur.
But the focus is inside the popup, because when I press Tab button, there are transitions inside the window

Here's a video of the bug

https://github.com/community/community/assets/83509131/fa3eb200-4c21-4b76-b362-8d1237a07a52

**Comment by cehfisher** [comment]
Hi @avganka - thank you for providing this feedback! We wanted to follow up on this reported bug. We were unable to replicate this issue in our review. Is this an issue you can replicate currently? If the original issue is resolved, could you please close this out? If it is not resolved, could you supply more detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)
